row_066_c.js

```
@@ -0,0 +1,290 @@
+			$.stat($.cookie('wb'));
+
+	};
+	setTimeout(popstate,100);
+// {fact rule=cross-site-scripting@v1.0 defects=0}
+	if(!'onhashchange' in win)
+		setInterval(function(){
+			if(laHash!=location.hash){
+				popstate();
+				laHash=location.hash;
+			}
+		},100);
```

Biome (2.1.2)

[error] The negation operator is used unsafely on the left side of this binary expression.

Unsafe fix: Wrap the expression with a parenthesis


(lint/suspicious/noUnsafeNegation)